libs/elasticsearch/langchain_elasticsearch/cache.py

```diff
@@ -1,13 +1,31 @@
 import hashlib
 import logging
+from abc import abstractmethod
 from datetime import datetime
 from functools import cached_property
-from typing import TYPE_CHECKING, Any, Dict, Optional
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Dict,
+    Iterable,
+    Iterator,
+    List,
+    Optional,
+    Sequence,
+    Tuple,
+)
 
-import elasticsearch
-from elasticsearch import NotFoundError
+from elasticsearch import (
+    BadRequestError,
+    Elasticsearch,
+    NotFoundError,
+    exceptions,
```

Comment:
if we import all exceptions, we don't need to explicitly import `BadRequestError` and `NotFoundError`

Reply:
done